packages/web-app-epub-reader/tests/unit/app.spec.ts

```diff
@@ -1,23 +1,197 @@
+        expect((wrapper.find(selectors.increaseFontSize) as any).isDisabled()).toBe(true)
+      })
+    })
+    describe('decrease font size button', () => {
+      it('decreases font size when clicked', async () => {
+        const { wrapper } = getWrapper()
+        await nextTicks(2)
+        await wrapper.find(selectors.decreaseFontSize).trigger('click')
+        expect(wrapper.vm.currentFontSizePercentage).toBe(90)
+      })
+      it('is disabled when "MIN_FONT_SIZE_PERCENTAGE" is reached', () => {
+        const { wrapper } = getWrapper({ localStorage: { fontSizePercentage: 50 } })
+        expect((wrapper.find(selectors.decreaseFontSize) as any).isDisabled()).toBe(true)
+      })
+    })
+    describe('reset font size button', () => {
+      it('resets font size when clicked', async () => {
+        const { wrapper } = getWrapper({ localStorage: { fontSizePercentage: 50 } })
+        await nextTicks(2)
+        await wrapper.find(selectors.resetFontSize).trigger('click')
+        expect(wrapper.vm.currentFontSizePercentage).toBe(100)
+      })
+      it('shows the current font size', async () => {
+        const { wrapper } = getWrapper()
+        await nextTicks(2)
+        await wrapper.find(selectors.decreaseFontSize).trigger('click')
+        expect(wrapper.find(selectors.resetFontSize).text()).toBe('90%')
+      })
+    })
+  })
+  describe('chapters', () => {
+    describe('chapters list', () => {
+      it('renders correctly', async () => {
+        const { wrapper } = getWrapper()
+        await nextTicks(2)
+        const chapterElements = wrapper.findAll(selectors.chaptersListItem)
+        expect(chapterElements.length).toEqual(2)
+        expect(chapterElements[0].text()).toEqual('Chapter 1')
+        expect(chapterElements[1].text()).toEqual('Chapter 2')
+      })
+      it('calls method "display" when item is clicked', async () => {
+        const { wrapper } = getWrapper()
+        await nextTicks(2)
+        const chapterElements = wrapper.findAll(selectors.chaptersListItem)
+        await chapterElements[1].find('.oc-button').trigger('click')
+        expect(wrapper.vm.rendition.display).toHaveBeenCalledWith('c2')
+      })
+    })
+    describe('chapters select', () => {
+      it('renders correctly', async () => {
+        const { wrapper } = getWrapper()
+        await nextTicks(2)
+        const chapterElements = await getOcSelectOptions(wrapper, selectors.chaptersSelect)
+        expect(chapterElements.length).toEqual(2)
+        expect(chapterElements[0].text()).toEqual('Chapter 1')
+        expect(chapterElements[1].text()).toEqual('Chapter 2')
+      })
+      it('calls method "display" when item is clicked', async () => {
+        const { wrapper } = getWrapper()
+        await nextTicks(2)
+        const chapterElements = await getOcSelectOptions(wrapper, selectors.chaptersSelect)
+        await chapterElements[1].trigger('mousedown')
+        expect(wrapper.vm.rendition.display).toHaveBeenCalled()
```

Comment:
Shouldn't you assert the param here as well?

Reply:
Hehe yes but it didn't work. Too bad you are after me 🤓